<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UCalculationChannelsWidget</class>
 <widget class="QWidget" name="UCalculationChannelsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>103</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidgetChannels">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Channel</string>
   </property>
  </action>
  <action name="actionInsertChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Insert Channel</string>
   </property>
   <property name="toolTip">
    <string>Insert Channel</string>
   </property>
  </action>
  <action name="actionDeleteSelectedChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete Selected Channel</string>
   </property>
  </action>
  <action name="actionCloneChannel">
   <property name="text">
    <string>Clone Channel</string>
   </property>
  </action>
  <action name="actionStartChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Channel</string>
   </property>
  </action>
  <action name="actionPauseChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause Channel</string>
   </property>
  </action>
  <action name="actionResetChannel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset Channel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
